### ISSUE TITLE
The remote app is not working.

### ISSUE THREAD

**Issue by Gaboorz** [ciderapp/Cider-2#1326]
### Preflight Checklist

- [x] I agree to follow the [Code of Conduct](https://cider.sh/legal/conduct) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/ciderapp/Cider-2/issues) for a bug report that matches the one I want to file, without success.
- [x] I have confirmed that no [support FAQ](https://cider.sh/docs/client/faqs) resolves my issue.
- [x] I have checked that the correct behavior occurs on [Apple Music Web](https://music.apple.com) (if applicable).
- [x] I have verified that my issue is with a [supported version](https://github.com/ciderapp/Cider-2/blob/main/SECURITY.md) and that it is with the desktop app (Support for other areas can be found in our Discord / on the appropriate repo).

### How did you download Cider?

Microsoft Store

### App Information

{"clientInfo":{"client":"dotnet","env":"production","arch":"AMD64","platform":"win32","platformFriendly":"Windows","build":"msstore","clientVersion":"310","capi":"d84f395e-ffac-4f83-9283-9e9a1f24374b","subversion":"1"},"buildInfo":{"version":"3.1.10","commit":"9f74e53bbb8573bde9deaad4f90f91014983b9e2","branch":"main","buildDate":"1766700364604","changelogUrl":"3.1.0-release","codeName":"Gala"}}

### Operating System

Windows

### OS Version

Windows 11

### How would you describe your issue?

Accessibility

### Commit Hash

9f74e53bbb8573bde9deaad4f90f91014983b9e2

### Describe the Bug

When I try to use the remote app, it doesn't work and it says that the device is not playing anything.

### Steps to Reproduce

It happened to me when I was trying to connect my PC to my iPhone.
